.taskcluster.yml

```diff
@@ -1,6 +1,51 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
 version: 0
 allowPullRequests: collaborators
 tasks:
+####################################################################################################
+# Task: Pull requests
+####################################################################################################
+  - provisionerId: '{{ taskcluster.docker.provisionerId }}'
+    workerType: '{{ taskcluster.docker.workerType }}'
+    extra:
+      github:
+        env: true
+        events:
+          - pull_request.opened
+          - pull_request.edited
+          - pull_request.synchronize
+          - pull_request.reopened
+          - push
+    scopes:
+      - "queue:create-task:aws-provisioner-v1/github-worker"
+      - "queue:scheduler-id:taskcluster-github"
+    payload:
+      maxRunTime: 3600
+      deadline: "{{ '2 hours' | $fromNow }}"
+      image: 'mozillamobile/android-components:1.4'
+      command:
+        - /bin/bash
+        - '--login'
+        - '-cx'
+        - >-
+          export TERM=dumb
+          && git fetch {{ event.head.repo.url }} {{ event.head.repo.branch }}
+          && git config advice.detachedHead false
+          && git checkout {{ event.head.sha }}
+          && python automation/taskcluster/decision_task_pull_request.py
+      features:
+        taskclusterProxy: true
```

Comment:
I assume this is needed too :)

Reply:
Well, sort of.  Without the TC proxy, you can't download secrets.  We don't use that _yet_, but we will if and when we want to publish Android libraries in release tasks.